### PR TITLE
Position details data point

### PR DESCRIPTION
**Pull request by mjoyce91**
Adds a more granular component for the data points inside the PositionDetailsItem component. Based on feedback from @burgwyn in https://github.com/18F/State-TalentMAP/pull/301

### CODE REVIEW

File: src/Components/PositionDetailsDataPoint/PositionDetailsDataPoint.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const PositionDetailsDataPoint = ({ title, description }) => (
+  <div className="usa-width-one-fourth position-details-description-section">
+    <div className="title">{title}</div>
+    <div className="description">{description}</div>
+  </div>
+);
+
+PositionDetailsDataPoint.propTypes = {
+  title: PropTypes.string.isRequired,
+  description: PropTypes.oneOfType([
```
Comment (burgwyn) [inline src/Components/PositionDetailsDataPoint/PositionDetailsDataPoint.jsx]: I like what you're thinking, maybe we go with `PropTypes.node`? - https://facebook.github.io/react/docs/typechecking-with-proptypes.html

File: src/Components/PositionDetailsDataPoint/PositionDetailsDataPoint.test.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  });
+
+  it('handles numeric props', () => {
+    wrapper = shallow(
+      <PositionDetailsDataPoint title="Title" description={1} />,
+    );
+    expect(wrapper.instance().props.description).toBe(1);
+  });
+
+  it('handles element props', () => {
```
Comment (burgwyn) [inline src/Components/PositionDetailsDataPoint/PositionDetailsDataPoint.test.jsx]: If we do `PropTypes.node` then we may want to add some other types

**Comment by mjoyce91** [comment]
Changed prop type to `node` and updated tests accordingly.

File: src/Components/PositionDetailsDataPoint/PositionDetailsDataPoint.test.jsx [hunk 3]
```diff
@@ -35,4 +35,12 @@ describe('PositionDetailsDataPoint', () => {
     );
     expect(wrapper.instance().props.description).toBe(description);
   });
+
+  it('handles an array of different prop types', () => {
```
Comment (burgwyn) [inline src/Components/PositionDetailsDataPoint/PositionDetailsDataPoint.test.jsx]: ‼️

**Pull request merged**